server/app/views/admin/programs/ProgramBlockBaseView.java

```diff
@@ -94,11 +83,17 @@ protected final DivTag renderExistingPredicate(
     return container.with(conditionList);
   }
 
-  /** Returns the string that will be shown on the Edit button */
-  protected abstract String getEditButtonText();
-
-  /** Returns the navigation destination for the Edit button */
-  protected abstract String getEditButtonUrl(ProgramDefinition programDefinition);
+  /**
+   * Returns a standardized Edit Button that can be added to the program info. A typical use case
+   * would be for a subclass to - create a button with this method - add subclass specific
```

Comment:
**suggestion (blocking):** I think these are meant to be bullet points? Could you update the formatting if so?